data-serving/data-service/api/dataserver.api.yaml

```diff
@@ -0,0 +1,54 @@
+openapi: 3.0.3
+info:
+  title: EPID data service
+  description: Server for CRUD operations on EPID line-list data
+  termsOfService: 'https://www.healthmap.org/about/termsofuse/'
+  contact:
+    email: info@healthmap.org
+  license:
+    name: MIT
+  version: 1.0.0
+paths:
+  /api/cases/{id}:
+    summary: Line-list cases
+    get:
+      summary: Gets a specific case
+      parameters: 
+        - in: path
+          name: id
+          required: true
+          schema:
+            type: string
+          description: The case ID
+      responses:
+        '200':
+          description: Case found.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Case'
+        '400':
```

Comment:
This API (perhaps mistakenly) returns 422 for all bad requests.

Reply:
hmm... I'll add 422 (and keep 400 for malformed requests)